backend/samfundet/views.py

```diff
@@ -712,6 +712,19 @@ def list(self, request: Request) -> Response:
         return Response(serializer.data)
 
 
+class RecruitmentAdmissioWithdrawApplicantView(APIView):
```

Comment:
```suggestion
class RecruitmentAdmissionWithdrawApplicantView(APIView):
```
Du kan resolve denne når det er endret

Reply:
Done

Reply:
Løste også det med den under, setter bare en if statement rundt